.travis.yml

```diff
@@ -19,7 +19,7 @@ install:
   - git clone https://github.com/jdebacker/Tax-Calculator.git && cd Tax-Calculator
   - git fetch --all && git checkout master && python setup.py install
```

Comment:
change the highlighted line from:
`git fetch --all && git checkout master && python setup.py install`
to
`git fetch --all && git checkout master && python setup.py install && cd ../B-Tax`